webpack.development.config.js

```diff
@@ -115,6 +115,7 @@ module.exports = {
     ]),
     new BundleAnalyzerPlugin({
       analyzerHost: '0.0.0.0',
+      analyzerPort: '8888'
```

Comment:
Oops, I was trying to get the coverage to run while the app was running here -- can remove